[fix #4] [mozmeao/infra#1104] Clean up oregon-b dev yaml.

jpetto opened the pull request:
- rename svc.yaml to web-svc.yaml

jpetto committed:
[fix #4] [mozmeao/infra#1104] Clean up oregon-b dev yaml.

- rename svc.yaml to web-svc.yaml

jpetto committed:
update image hashes

jpetto committed:
add k8s namespace/podname

jgmize commented on oregon-b/basket-dev/donateworker-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-              key: wsgi-num-workers
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-dev/donateworker-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-dev/ns.yaml:
```diff
@@ -1,13 +1,7 @@
-  resourceVersion: "1120334"
-  selfLink: /api/v1/namespaces/basket-dev
-  uid: 857dddb5-0152-11e8-8c8b-0656fabf84c0
 spec:
```
spec and status sections can be removed too

jgmize commented on oregon-b/basket-dev/web-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-dev-v125-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-dev/web-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-dev-v125-env
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-b/basket-dev/web-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-dev-v125-env
+          value: 0
```
```suggestion
          value: "0"
```

jgmize commented on oregon-b/basket-dev/web-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-              key: wsgi-num-workers
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-dev/web-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-dev/worker-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-dev-v125-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-dev/worker-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-dev-v125-env
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-b/basket-dev/worker-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-dev-v125-env
+          value: 0
```
```suggestion
          value: "0"
```

jgmize commented on oregon-b/basket-dev/worker-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-              key: wsgi-num-workers
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-dev/worker-deploy.yaml:
```diff
@@ -48,376 +37,231 @@ spec:
-              name: basket-dev-v125-env
-        image: 127.0.0.1:5555/basket-dev:v125
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize reviewed (changes requested):
see comments on https://github.com/mozmeao/basket-config/pull/20 -- we need to make the same changes here and in the other PRs

jpetto committed:
Review fixes:

- remove superfluous info from ns.yaml
- quote all numeric env var values
- use full image hash

jpetto commented:
@jgmize updated!

jgmize commented:
thanks again @jpetto!
